### PR TITLE
fixed bug on desktop when no versions are installed

### PR DESCRIPTION
**Pull request by maxcapodi78**

### CODE REVIEW

File: pyaedt/application/Analysis.py [hunk 1]
```diff
@@ -1652,20 +1652,24 @@ def submit_job(
...
+                    r"\\\\\\\\" + clustername + r"\\\\AnsysEM\\\\{}\\\\Linux64\\\\ansysedt".format(version_name)
                 )
             else:
-                self.logger.error("AEDT path does not exist. Please provide a full path.")
+                self.logger.error("AEDT shared path does not exist. Please provide a full path.")
                 return False
         else:
             if not os.path.exists(aedt_full_exe_path):
-                self.logger.error("Aedt Path doesn't exists. Please provide a full path")
+                self.logger.error("Aedt shared Path doesn't exists. Please provide a full path")
```
Comment (PipKat) [inline pyaedt/application/Analysis.py:1672]:
```suggestion
                self.logger.error("AEDT shared path does not exist. Provide a full path.")
```

File: pyaedt/generic/general_methods.py [hunk 2]
```diff
@@ -1062,6 +1062,7 @@ def __init__(self):
         self.formatter = None
         self.remote_rpc_session = None
         self.remote_rpc_session_temp_folder = ""
+        self.remote_rpc_service_manager_port = 17878
```
Comment (MaxJPRey) [inline pyaedt/generic/general_methods.py:1065]: @maxcapodi78 Was it working previously without hard-coding the port?

**Review by MaxJPRey — APPROVED** [review]

File: pyaedt/generic/general_methods.py (hunk 2 above)
Comment (maxcapodi78) [reply pyaedt/generic/general_methods.py:1065]: before was hardcoded. now it's in settings and can be changed

Comment (MaxJPRey) [reply pyaedt/generic/general_methods.py:1065]: LGTM.

**Pull request merged**